sections/main-addresses.liquid

```diff
@@ -27,92 +36,125 @@
-          </div>
-          <div class="field">
+          <label for="AddressCompanyNew">{{ 'customer.addresses.company' | t }}</label>
+        </div>
+        <div class="field">
             <input type="text" id="AddressAddress1New" name="address[address1]" value="{{ form.address1 }}" autocomplete="address-line1" placeholder="{{ 'customer.addresses.address1' | t }}">
-            <label for="AddressAddress1New">{{ 'customer.addresses.address1' | t }}</label>
-          </div>
-          <div class="field">
+          <label for="AddressAddress1New">{{ 'customer.addresses.address1' | t }}</label>
+        </div>
+        <div class="field">
             <input type="text" id="AddressAddress2New" name="address[address2]" value="{{ form.address2 }}" autocomplete="address-line2" placeholder="{{ 'customer.addresses.address2' | t }}">
-            <label for="AddressAddress2New">{{ 'customer.addresses.address2' | t }}</label>
-          </div>
-          <div class="field">
+          <label for="AddressAddress2New">{{ 'customer.addresses.address2' | t }}</label>
+        </div>
+        <div class="field">
             <input type="text" id="AddressCityNew" name="address[city]" value="{{ form.city }}" autocomplete="address-level2" placeholder="{{ 'customer.addresses.city' | t }}">
-            <label for="AddressCityNew">{{ 'customer.addresses.city' | t }}</label>
+          <label for="AddressCityNew">{{ 'customer.addresses.city' | t }}</label>
+        </div>
+        <div class="field-country">
+          <label for="AddressCountryNew">{{ 'customer.addresses.country' | t }}</label>
+          <div class="select">
+            <select
+              id="AddressCountryNew"
+              name="address[country]"
+              data-default="{{ form.country }}"
+              autocomplete="country"
+              {% if form.errors contains 'country' %}
+                aria-invalid="true"
+                aria-describedby="AddressCountryNewError"
+              {% endif %}
+            >
+              {{ all_country_option_tags }}
+            </select>
+            <svg aria-hidden="true" focusable="false" viewBox="0 0 10 6">
+              <use href="#icon-caret" />
+            </svg>
           </div>
-          <div>
-            <label for="AddressCountryNew">{{ 'customer.addresses.country' | t }}</label>
-            <div class="select">
-              <select
-                id="AddressCountryNew"
-                name="address[country]"
-                data-default="{{ form.country }}"
-                autocomplete="country"
+        </div>
+        {%- if form.errors contains 'country' -%}
+          <span id="AddressCountryNewError" class="form__message">
+            <svg aria-hidden="true" focusable="false">
+              <use href="#icon-error" />
+            </svg>
+            {{ form.errors.translated_fields['country'] | capitalize }} {{ form.errors.messages['country'] }}.
```

Comment:
I think this is where live regions would be helpful, because if we announce the error as soon as it appears under the input, then we don't need to also summarise things at the top (for short forms, anyway).